### DISCUSSION TITLE
webstorm login copilot 401

### DISCUSSION THREAD

**Discussion by zhoufanglu** [community/community#147791] [Copilot Conversations]
### Select Topic Area

Question

### question
Sign in failed. Reason: Request signInInitiate failed with message: Response content-type is text/plain; charset=utf-8 (status=401), request id: 151, error code: -32603
![image](https://github.com/user-attachments/assets/be5b64ca-486d-4c11-96a1-6dbb12640957)

### version
- WebStorm 2024.3.1
- github copilot 1.5.30-242

**Comment by itsZORAL1** [comment]
It looks like you're getting a 401 error when trying to log into GitHub Copilot in WebStorm. This can happen if there’s an issue with the login or authentication. I’d recommend signing out and signing back in, making sure your GitHub credentials are correct. You could also try clearing your cache or checking if a proxy is causing issues. Also, make sure both WebStorm and GitHub Copilot are up to date. Hope that helps!

**Comment by mooosakhan** [comment]
It looks like you're getting a 401 error when trying to log into GitHub Copilot in WebStorm. This can happen if there’s an issue with the login or authentication. I’d recommend signing out and signing back in, making sure your GitHub credentials are correct. You could also try clearing your cache or checking if a proxy is causing issues. Also, make sure both WebStorm and GitHub Copilot are up to date. Hope that helps!

**Comment by zhoufanglu** [comment]
1. I reinstalled my webstorm
2. I cleared all my configurations and did not import my previous setting.zip. I do not know exactly which configuration item it is, and then it ran successfully
